support-frontend/assets/helpers/contributions.ts

```diff
@@ -407,108 +393,6 @@ function billingPeriodFromContrib(
 	}
 }
 
-function errorMessage(
-	error: ContributionError,
-	contributionType: ContributionType,
-	countryGroupId: CountryGroupId,
-): string | null | undefined {
-	const minContrib = config[countryGroupId][contributionType].min;
-	const maxContrib = config[countryGroupId][contributionType].max;
-	const currency = currencies[countryGroups[countryGroupId].currency];
-
-	switch (error) {
-		case 'TooLittle':
-			return `Please enter at least ${currency.glyph}${minContrib}`;
-
-		case 'TooMuch':
-			return `${currency.glyph}${maxContrib} is the maximum we can accept`;
-
-		case 'ParseError':
-			return 'Please enter a numeric amount';
-
-		default:
-			return null;
-	}
-}
-
-function getContributionTypeClassName(
-	contributionType: ContributionType,
-): string {
-	if (contributionType === 'ONE_OFF') {
-		return 'one-off';
-	} else if (contributionType === 'ANNUAL') {
-		return 'annual';
-	}
-
-	return 'monthly';
-}
-
-function getSpokenType(contributionType: ContributionType): string {
-	if (contributionType === 'ONE_OFF') {
-		return 'single';
-	} else if (contributionType === 'ANNUAL') {
-		return 'annual';
-	}
-
-	return 'monthly';
-}
-
-function getFrequency(contributionType: ContributionType): string {
-	if (contributionType === 'ONE_OFF') {
-		return '';
-	} else if (contributionType === 'MONTHLY') {
-		return 'per month';
-	}
-
-	return 'per year';
-}
-
-function getCustomAmountA11yHint(
```

Comment:
I wonder if this would be a useful thing to dig into where it was used and potentially add to our checkout again?

Reply:
Yeah perhaps worth adding a card for (to investigate and fix if needed)?